draft-ietf-tls-rfc8447bis.md

```diff
@@ -535,6 +535,19 @@ IANA is requested to add a "Comment" column to the following registries:
 This list of registries is all registries that do not already have a
 "Comment" or "Notes" column or that were not orphaned by TLS 1.3.
 
+# Expert Review of Current and Potential IETF and IRTF Documents
+
+The intent of the Specification Required standard for TLS code points
+is to allow for easy registration for code points associated with
+protocols and algorithms that are not being actively developed inside
+IETF or IRTF. When TLS-based technologies are being developed inside
+the IRTF/IETF they should be done in coordination with the TLS WG in
+order to provide appropriate review. For this reason, designated
+experts should discourage code point registrations for documents which
+have already been adopted or are being proposed for adoption by IETF
+working groups or IRTF research groups.
```

Comment:
I agree with Rich that we should provide a bit more clarity on our expectations about "discourage" in the face of resistance to the discouragement.

Reply:
Perhaps I was being too cute and we should simply say "decline".

Reply:
My mental model is that we do want to leave some room for the expert judgment of the experts, but would prefer phrasing involving "decline" to phrasing involving "discourage".

Reply:
Isn't the "should" what leaves room?

Reply:
I am one of three DE's for the TLS registries. I don't know how Nick and Yoav feel. But i greatly prefer to have as little wiggle-room as possible, if only because there is no formal process for a WG, the IESG, or the IETF community to "overrule" a DE decision, short of replacing them which seems a little over-the-top.